components/eam/src/chemistry/modal_aero/modal_aero_amicphys.F90

```diff
@@ -5498,6 +5549,12 @@ subroutine modal_aero_amicphys_init( imozart, species_class,n_so4_monolayers_pca
       nait = modeptr_aitken
       npca = modeptr_pcarbon
       nufi = modeptr_ufine
+
+      ! these lines turn on (ncrs>0) or off (ncrs=0) acc->crs renaming in stratosphere
+      if (strat_accum_coarse_rename) then
+         ncrs = modeptr_coarse
```

Comment:
This looks like the only place `ncrs` gets set, so if `strat_accum_coarse_rename` is false, then it looks like `ncrs` is uninitialized (and thus I think not guaranteed to be zero).

Reply:
That makes sense. I overlooked this when I added switch to turn this on. I will initialize it to zero with the other pointers (nacc, nait, npca, nufi) on the lines above this instance.

Reply:
Added initialization of ncrs above logical